translations/dde-store_ar.ts

```diff
@@ -0,0 +1,537 @@
+
+        </message>
+
+    </context>
+
+    <context>
+
+        <name>UpdatesPage</name>
+
+        <message>
+
+            <location filename="../pages/updatespage.cpp" line="12"/>
+
+            <source>Updates</source>
+
+            <translation>التحديثات</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../pages/updatespage.cpp" line="13"/>
+
+            <source>Up to date</source>
+
+            <translation>لا يوجد تحديثات</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../pages/updatespage.cpp" line="15"/>
+
+            <source>System Updates</source>
+
+            <translation>تحديثات النظام</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../pages/updatespage.cpp" line="27"/>
+
+            <source>Update All</source>
+
+            <translation>جميع التحديثات</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../pages/updatespage.cpp" line="50"/>
+
+            <source>Updating...</source>
+
+            <translation>يتم التحديث...</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../pages/updatespage.cpp" line="53"/>
+
+            <source>Update All (%1)</source>
+
+            <translation>تحديث جميع (%1)</translation>
+
+        </message>
+
+    </context>
+
+    <context>
+
+        <name>stars</name>
+
+        <message>
+
+            <location filename="../widgets/stars.cpp" line="23"/>
+
+            <source>(%1 ratings)</source>
+
+            <translation>(%1 تقييم)</translation>
+
+        </message>
+
+    </context>
+
+    <context>
+
+        <name>AppStreamHelper</name>
+
+        <message>
+
+            <location filename="../backend/appstreamhelper.cpp" line="28"/>
+
+            <source>ERROR: Unable to open AppStream metadata pool</source>
+
+            <translation>خطأ: تعذر فتح مستودع معلومات AppStream</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../backend/appstreamhelper.cpp" line="32"/>
+
+            <source>OK</source>
+
+            <translation>موافق</translation>
+
+        </message>
+
+    </context>
+
+    <context>
+
+        <name>PackageKitHelper</name>
+
+        <message>
+
+            <location filename="../backend/packagekithelper.cpp" line="59"/>
+
+            <source>Updates Available</source>
+
+            <translation>يتوفر تحديثات جديدة</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../backend/packagekithelper.cpp" line="120"/>
+
+            <source>Installed \"%1\"</source>
+
+            <translation>تم تثبيت\"%1\"</translation>
+
+        </message>
+
+        <message>
+
+            <location filename="../backend/packagekithelper.cpp" line="140"/>
+
+            <source>Uninstalled \"%1\"</source>
+
+            <translation>تم إزالة تثبيت \"%1\"</translation>
```

Comment:
\"%1\" should be treated the same as %1. In this case, \" was moved to the end and after %1. Make sure it stays together as \"%1\"

Reply:
It looks different because the text is RTL. In reality, it's the exact same thing. Try to copy it to check or just try Ctrl+F to find it.

Reply:
Ohh, sorry, you're right. Thanks for clarifying.

Reply:
Since I'm a Paleastenian, the Paleastenian flag would be my choice. If that could (or you think it might) cause an issue, you can use the flag of the Arab League. If that's not an option, you may use the flag of Saudi Arabia.